src/content/blog/images.mdx

```diff
@@ -10,7 +10,7 @@ coverImage: "/src/content/blog/_images/images/blog-hero-image-updates.webp"
 socialImage: "/src/content/blog/_images/images/og-image-image-better.jpg"
 ---
 
-import { Image } from "@astrojs/image/components"
+import { Image } from "astro:assets"
```

Comment:
Revert this one, looks like a find/replace caught the blog post's example of the old integration

Reply:
This one was a functional import, there were other ones _elsewhere_ that I updated instead